setup.cfg

```diff
@@ -14,6 +14,6 @@ warn_unused_configs = True
 check_untyped_defs = True
 disallow_untyped_defs = True
 [tool:pytest]
-addopts = -p no:playwright -p no:playwright-asyncio --runpytest subprocess -vv
+addopts = -p no:asyncio -p no:playwright -p no:playwright-asyncio --runpytest subprocess -vv
```

Comment:
drive-by: We don't need `pytest-asyncio` in our main test-suite. Only in the sub-tests we need it.